### ISSUE TITLE
at (@) keyboard shortcut does not work on mac with swiss german keyboard layout

### ISSUE THREAD

**Issue by aaronschmied** [atom/atom#9945]
In the swiss german mac keyboard layout, the at (@) sign is made using the `ALT-G` keyboard shortcut. In atom, this does not work, which makes it very inconvenient to code.

**Comment by DavidAbbe** [comment]
Try this:
Go to File>Settings>Packages and search for the autoflow Package and disable it. After restarting Atom it should work.

**Comment by aaronschmied** [comment]
Did not make any difference. I still can't write any "@"s in atom.

**Comment by aaronschmied** [comment]
I found a solution to my problem:

Two packages were using the `ALT-G` keybindings. *git-diff* and *open-on-github*. In the git-diff package settings i could deactivate the keybindings, so this was not a problem. The open-on-github package on the other hand did not have such an option, so I deactivated it. After a restart of Atom, it worked perfectly.

This could be still an issue for other users with the swiss german mac keyboard layot.
